### PR TITLE
Feature/pkgmgmtdoc

### PR DESCRIPTION
**Pull request by zfengms**
Merge package management docs in 2 places.

### CODE REVIEW

File: docs/20-package-management.md [hunk 1]
```diff
@@ -4,7 +4,19 @@ The doAzureParallel package allows you to install packages to your pool in two w
...
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
+
 ## Installing Packages on Pool Creation
+
+Pool level packages support CRAN, GitHub and BioConductor packages. The packages are installed in a shared directory on the node. It is important to note that it is required to explicitly load any packages installed at the cluster level within the foreach loop. For example, if you installed xml2 on the cluster, you must explicityly load it before using it.
+
+```R
+foreach (i = 1:4) %dopar% {
+  # Load the libraries you want to use.
+  library(xml2)
```
Comment (brnleehng) [inline docs/20-package-management.md]: Is this true? I thought you can do .packages() and it'll automatically be called.

File: samples/package_management/README.md [hunk 2]
```diff
@@ -1,69 +1,3 @@
...
-
-### CRAN
-
-```R
-foreach(i = 1:4, .packages = c("xml2")) %dopar% {
-  # xml2 is automatically loaded an can be used without calling library(xml2)
-  xml2::as_list(...)
-}
-```
+Please see documentation[(link)](../../docs/20-package-management.md) for more details on packagement management.
```
Comment (brnleehng) [inline samples/package_management/README.md]: Remove this. And just add a comment line in the sample

File: docs/20-package-management.md [hunk 3]
```diff
@@ -4,7 +4,19 @@ The doAzureParallel package allows you to install packages to your pool in two w
 - Installing on pool creation
 - Installing per-*foreach* loop
 
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
+
 ## Installing Packages on Pool Creation
+
+Pool level packages support CRAN, GitHub and BioConductor packages. The packages are installed in a shared directory on the node. It is important to note that it is required to explicitly load any packages installed at the cluster level within the foreach loop. For example, if you installed xml2 on the cluster, you must explicityly load it before using it.
```
Comment (brnleehng) [inline docs/20-package-management.md]: typo: explicitly

File: docs/20-package-management.md [hunk 4]
```diff
@@ -4,7 +4,19 @@ The doAzureParallel package allows you to install packages to your pool in two w
 - Installing on pool creation
 - Installing per-*foreach* loop
 
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
```
Comment (paselem) [inline docs/20-package-management.md]: "benefit from specifying any specific" is a weird sentence. May be "benefit from specifying any dependencies"

File: docs/20-package-management.md (hunk 1 above)
Comment (paselem) [reply docs/20-package-management.md]: @brnleehng - I thought that was only true for job-level packages, right? For cluster level packages I think we need to either explicitly load the library or do __xml2::__

File: docs/20-package-management.md (hunk 4 above)
Comment (zfengms) [reply docs/20-package-management.md]: fixed.

File: docs/20-package-management.md (hunk 1 above)
Comment (brnleehng) [reply docs/20-package-management.md]: For the job level, we will try to reinstall and reference it with library. For the cluster level, we would need xml2::

**Pull request merged**